08.ls_object/ls.rb

```diff
@@ -7,13 +7,24 @@
 require_relative 'long_format'
 
 class Ls
+  def self.main
+    options = {}
+    opt = OptionParser.new
+    opt.on('-a') { |v| options[:a] = v }
+    opt.on('-l') { |v| options[:l] = v }
+    opt.on('-r') { |v| options[:r] = v }
+    target = opt.parse(ARGV)[0] || '.'
+
+    puts Ls.new(target, options).show
```

Comment:
`self == Ls`なのでnewだけでよかったりします。

```suggestion
    puts new(target, options).show
```

Reply:
修正しました！ 361a1bf